src/main/resources/datical/datical_deploy.bat.ftl

```diff
@@ -10,4 +10,5 @@
 
 -->
 <#include "/datical/datical_generic.ftl">
-${login} -p ${deployed.targetPath} deploy ${environment} ${labels} ${reports} ${pipeline}
+<#include "/datical/datical_credentials.bat.ftl">
+${login_simple} -p ${deployed.targetPath} deploy ${environment} ${labels} ${reports} ${pipeline}
```

Comment:
A better idea is to replace ```login``` with ```hammer``` because now that we have taken out database credentials and are no longer needed with the ```hammer``` command.